<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminpage</class>
 <widget class="QMainWindow" name="adminpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="adminHomePage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADMIN PAGE</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>551</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>191</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addCampusThroughInputFile">
         <property name="text">
          <string>Add Campus (Input File)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addSouvenirThroughInputFile">
         <property name="text">
          <string>Add Souvenir (Input File)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addSouvenirBtn">
         <property name="text">
          <string>Add Souvenir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteSouvenirBtn">
         <property name="text">
          <string>Delete Souvenir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>460</y>
        <width>421</width>
        <height>36</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="viewDistancesBtn">
         <property name="text">
          <string>View Distances</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewSouvenirBtn">
         <property name="text">
          <string>View Souvenir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeSouvenirPriceBtn">
         <property name="text">
          <string>Change Price</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="deleteSouvenirPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete Souvenir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteSouvenirConfirmBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>390</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Souvenir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletePageBackBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Please choose a Campus and Souvenir in order to delete</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="text">
        <string>Campus Name:</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
      <widget class="QComboBox" name="deletePageComboBox"/>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="text">
        <string>Souvenir Name:</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="deletePageSouvenirName_line"/>
     </widget>
    </widget>
    <widget class="QWidget" name="changeSouvenirPricePage">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Change Souvenir Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Please enter the Souvenir item in the Souvenir Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>box and update the price in the Souvenir Price box</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeSouvPriceBackBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateSouvenirPriceBtn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>500</y>
        <width>311</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Souvenir</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>115</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Campus Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nameCampuslbl_4">
         <property name="text">
          <string>Souvenir Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nameCampuslbl_5">
         <property name="text">
          <string>Souvenir Price:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QComboBox" name="changeSouvenirPageCampusNameComboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="souvPriceChangeName_line"/>
       </item>
       <item>
        <widget class="QLineEdit" name="souvChangePrice_line"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="addSouvenirPage">
     <widget class="QPushButton" name="addSouvenirPageBackBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>281</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="addSouvenirPageCampusNameComboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="souvName_line"/>
       </item>
       <item>
        <widget class="QLineEdit" name="souvPrice_line"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>200</y>
        <width>115</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Campus Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nameCampuslbl_2">
         <property name="text">
          <string>Souvenir Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nameCampuslbl_3">
         <property name="text">
          <string>Souvenir Price:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="createSouvenirBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>460</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Souvenir</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>374</width>
        <height>55</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Add Traditional Souvenir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Please add a Souvenir name and as well as the price</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
